reactive/k8s.py

```diff
@@ -31,11 +32,31 @@ def config_changed():
         docker_compose_kill_remove('proxy')
         remove_state('proxy.available')
 
-        if config.changed('version'):
-            hookenv.log('Removing kubectl.downloaded state so the new version'
-                        ' of kubectl will be downloaded.')
-            remove_state('kubectl.downloaded')
+    if config.changed('version'):
```

Comment:
the level of this if statement changed but this new level looks fine and will always evaluate.